releasenotes/notes/fix-profiling-native-mutices-62440b5a3d9d6c4b.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes a bug where profiling mutexes were not cleared on fork in the child process.  This could
+    cause deadlocks in certain configurations.
```

Comment:
```suggestion
    profiling: Fixes a bug where profiling mutexes were not cleared on fork in the child process.  This could
    cause deadlocks in certain configurations.
```